README.md

```diff
@@ -6,6 +6,8 @@ advantages reduces on large payloads.
 
 Benchmarks:
 
+Node 6.11.0:
+
```

Comment:
Sure! The I'll run it on my beefier work machine (15" 4 core instead of 13" 2 core)